### ISSUE TITLE
Doc error: dangling link in Define the commands that agents may use to traverse + modify a codebase (see here for more details)

### ISSUE THREAD

**Issue by didier-durand** [princeton-nlp/SWE-agent#384]
### Describe the bug

Hi,

The here in "(see here for more details)" on page https://github.com/princeton-nlp/SWE-agent/tree/main/config leads to a 404

Can you fix this link please?

### Steps/commands/code to Reproduce

```(see here for more details)```

### Error message/results

```404 http error (not found)```

### System Information

Github page https://github.com/princeton-nlp/SWE-agent/tree/main/config

### Checklist

- [X] I'm running with the latest docker container/on the latest development version
- [X] I've searched the other issues for a duplicate
- [X] I have copied the full command/code that I ran
- [X] I have copied the **full** log file/error message that was the result (if applicable)
- [X] I have enclosed code/log messages in triple backticks ([docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code)) and clicked "Preview" to make sure it's displayed correctly.

**Comment by didier-durand** [comment]
Hi,

Link under the here of "Define the commands that agents may use to traverse + modify a codebase (see [here](https://github.com/princeton-nlp/SWE-agent/blob/main/config/commands.md) for more details)" on https://github.com/princeton-nlp/SWE-agent/tree/main/config still gives 404

**Comment by klieret** [comment]
Oooh, now I see. You're clicking the links in the local readme files, which are symlinked to `docs/...`. 

Those won't work. Please use the [online documentation](https://princeton-nlp.github.io/SWE-agent/)

**Comment by didier-durand** [comment]
I AM using the online documentation. the problem that I report happens on the online reading of https://github.comprinceton-nlp/SWE-agent/tree/main/config.  The link under "here" (see details above) points to https://github.com/princeton-nlp/SWE-agent/blob/main/config/commands.md which doesn't exist.

**Comment by klieret** [comment]
Yes, this is file that is tracked in the repo. GitHub displays it and will resolve the relative link to an absolute URL (which doesn't exist). Please read the documentation at https://princeton-nlp.github.io/SWE-agent/
